src/main/java/org/jabref/logic/exporter/XmpExporter.java

```diff
@@ -0,0 +1,46 @@
+package org.jabref.logic.exporter;
+
+import java.io.BufferedWriter;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+import org.jabref.logic.util.FileType;
+import org.jabref.logic.xmp.XmpPreferences;
+import org.jabref.logic.xmp.XmpUtilWriter;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+
+/**
+ * A custom exporter to write bib entries to a .xmp file for further processing
+ * in other scenarios and applications. The xmp metadata are written in dublin
+ * core format.
+ */
+public class XmpExporter extends Exporter {
+
+    private final XmpPreferences xmpPreferences;
+
+    public XmpExporter(XmpPreferences xmpPreferences) {
```

Comment:
How do the other exporters get their preferences? Shouldn't they have the same issue with the Globals dependency?

Reply:
They get their prefs in the Exporter Factory. In this case it must only be expanded to pass the xmp prefs from the existing pref object